src/CommandingOfficers/CommanderCinder.java

```diff
@@ -0,0 +1,181 @@
+package CommandingOfficers;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import CommandingOfficers.Modifiers.COModifier;
+import Engine.XYCoord;
+import Engine.Combat.BattleSummary;
+import Engine.GameEvents.GameEventListener;
+import Terrain.GameMap;
+import Terrain.Location;
+import Units.Unit;
+import Units.UnitModel;
+
+/*
+ * Cinder is based on getting an edge in the action economy, at the cost of unit health.
```

Comment:
I think both of your new Commanders are definitely on the... evil side of the spectrum.

Reply:
I dunno, Bear&Bull skin their troops alive to turn a profit...

Reply:
I wasn't thinking of it as "skinning", but perhaps you have a point.